src/ac_copilot_trainer/modules/hud.lua

```
@@ -123,17 +126,50 @@ end
 function M.draw(vm)
   vm = vm or {}
+  if not _hudDiagLogged and ac and type(ac.log) == "function" then
+    _hudDiagLogged = true
+    -- Use tostring() so we see cdata/userdata/nil distinction (not just y/N)
+    local function tt(t, k)
+      if type(t) ~= "table" then return "?" end
+      local v = t[k]
+      if v == nil then return "nil" end
+      return type(v)
+    end
+    local szStr = "err"
```

Luacheck (1.2.0)

[warning] value assigned to variable 'szStr' is unused

(W311)